util/src/network/host.rs

```diff
@@ -170,29 +170,37 @@ pub struct NetworkContext<'s, Message> where Message: Send + Sync + Clone + 'sta
 	io: &'s IoContext<NetworkIoMessage<Message>>,
 	protocol: ProtocolId,
 	sessions: Arc<RwLock<Slab<SharedSession>>>,
-	session: Option<StreamToken>,
```

Comment:
Shared session pointer is used to output proper client id instead of "unknown" on disconnect and to guarantee that io handler is dealing with the same session.